CMakeLists.txt

```diff
@@ -1,13 +1,19 @@
 cmake_minimum_required(VERSION 2.8)
 
-project(snowhouse)
+project(include/snowhouse)
 
-option(SNOWHOUSE_BUILD_TESTS    "Build the Snowhouse tests"                 ON)
-option(SNOWHOUSE_RUN_TESTS      "Run the Snowhouse tests"                   ON)
+option(SNOWHOUSE_BUILD_TESTS    "Build the Snowhouse tests"                 OFF)
+option(SNOWHOUSE_RUN_TESTS      "Run the Snowhouse tests"                   OFF)
```

Comment:
Maybe it's because my lack of sleep, but I think you are talking about the AppVeyor configuration and I am referring to the Travis-CI-based CI ;-) I don't know why I have made that decision (there was probably a good reason for it), but the AppVeyor-based CI does not run the tests whereas the Travis-CI-based CI *did* run the tests and now it doesn't...

Compare the output for the [latest master branch](https://travis-ci.org/banditcpp/snowhouse/jobs/312057683) and for [your branch](https://travis-ci.org/banditcpp/snowhouse/jobs/437611603).

Reply:
Oh, I was just happy about getting a green light and tbh, I haven't really checked the logs. Will solve that.